lib/course_creation_manager.rb

```diff
@@ -9,8 +9,9 @@ class CourseCreationManager
 
   # rubocop:disable Metrics/ParameterLists
   def initialize(course_params, wiki_params, scoping_methods, initial_campaign_params,
-                 instructor_role_description, current_user)
+                 instructor_role_description, current_user, ta_support = false)
```

Comment:
Is a default to `false` necessary? If this only gets called from the controller, it should always have the parameter included, so leaving off the default value will help us catch any cases where it's being called differently.